amnestywebsite/humanity-theme
# Add spacing presets to theme.json

Ref: #334

This PR adds spacing presets to `theme.json` which creates preset css variables. 
SASS function `spacing()` is replaced with the `--wp--preset--spacing--*` variables, and all css files are replaced by preset variables.

**Steps to test**:
1. In the editor, verify the new spacing presets are available in the spacing tools.
<img width="277" alt="image" src="https://github.com/user-attachments/assets/0ebbcba6-67cf-47ce-828e-06baee9f7345">

2. Verify the theme and all available blocks. They should not get effected.

**Considerations**:
- It uses same schematics `quarter, half, single, double and quad`
- However it is recommended to use relative naming such as `x-small, small, medium` etc..
